pkgs/applications/science/math/giac/default.nix

```diff
@@ -104,7 +103,7 @@ stdenv.mkDerivation rec {
     description = "A free computer algebra system (CAS)";
     homepage = "https://www-fourier.ujf-grenoble.fr/~parisse/giac.html";
     license = licenses.gpl3Plus;
-    platforms = platforms.unix;
+    platforms = platforms.linux;
```

Comment:
Xcas is supposed to work on macos, and it used to (see #45364). cc @jbaum98 ?
In the worst case, it should be `platforms = if enableGui then linux else unix`.